src/Options/index.js

```diff
@@ -246,6 +246,10 @@ export interface ParseServerOptions {
   /* The security options to identify and report weak security settings.
   :DEFAULT: {} */
   security: ?SecurityOptions;
+  /* Set to true if new users should be created without public read and write access.
+  :ENV: PARSE_SERVER_ENFORCE_PRIVATE_USERS
```

Comment:
I don't think it is necessary to specify the ENV explicitly. It should be auto-generated from the option name. Could you check this by removing the ENV line and rebuild the config definitions?